src/client/components/Logout/Logout.story.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Logout from './Logout.component';
+
+storiesOf('Logouts', module).add('Logout Popup', () => <Logout />);
```

Comment:
Please update your storybook syntax. Check the guide in the readme file.

Reply:
ok